src/instrumentation/fetch.ts

```diff
@@ -57,7 +52,13 @@ export function gatherRequestAttributes(request: Request): Attributes {
 	const headers = request.headers
 	// attrs[SemanticAttributes.HTTP_CLIENT_IP] = '1.1.1.1'
 	attrs[SemanticAttributes.HTTP_METHOD] = request.method
-	attrs[SemanticAttributes.HTTP_URL] = sanitiseURL(request.url)
+	const u = new URL(request.url)
+	attrs[SemanticAttributes.HTTP_URL] = `${u.protocol}//${u.host}${u.pathname}${u.search}`
+	attrs[SemanticAttributes.HTTP_HOST] = u.host
+	attrs[SemanticAttributes.HTTP_SCHEME] = u.protocol
+	attrs[SemanticAttributes.HTTP_TARGET] = `${u.pathname}${u.search}`
+	// TODO parameterize route?
+	attrs[SemanticAttributes.HTTP_ROUTE] = `${u.pathname}`
```

Comment:
I don't think we should be using the pathname as the route. When you have REST APIs for example, the pathname will include IDs.